packages/@react-aria/gridlist/src/useGridListItem.ts

```diff
@@ -199,8 +246,9 @@ export function useGridListItem<T>(props: AriaGridListItemOptions, state: ListSt
     'aria-colindex': 1
   };
 
+  // TODO: should isExpanded and hasChildRows be a item state that gets returned by the hook?
```

Comment:
the above two states might be helpful for an aria implementation, but held off on adding them for now

Reply:
I still feel a bit hesitant to add them because I'm not really sure what an aria hooks only implementation of the expandable tree might look like and whether it may run into similar issues as our aria hooks only implementation of submenu trigger did (aka the aria implementation would use the old Collections implementation and Collection components and thus may run into https://github.com/adobe/react-spectrum/pull/5415#discussion_r1406568829). 

I imagine it wouldn't run into the same issue since we didn't have any issues with the expandable tableview implementation but I'd rather cross that bridge when we are actually work on the aria hooks only implementation and thus might find it helpful to expand the hooks further to return the above states. Maybe a bit of a hypocritiacal concern though since I am returning the aria attributes of the tree grid here haha.